cvts/settings.py

```diff
@@ -1,9 +1,15 @@
 import os
 import logging
+from enum import Enum
 from datetime import datetime as dt
 
+class RawDataFormat(Enum):
```

Comment:
Mongo is no more. Is there any reason we would keep it?

Reply:
The code still supports it. My understanding is that the data resides inside mongo inside Hanel. Depending on where this system winds up, it might still be useful.

If it became hard to maintain, then I'd remove it, but for now it is more work to remove it than keep it.

Reply:
I don't think Hanel uses it, but your point on removing only if it becomes hard to remove makes sense